core/templates/dev/head/services/contextual/UrlService.ts

```diff
@@ -131,6 +131,15 @@ angular.module('oppia').factory('UrlService', ['$window', function($window) {
     getOrigin: function() {
       return this.getCurrentLocation().origin;
     },
+    getCollectionIdFromExplorationUrl: function() {
+      if (this.getCurrentQueryString().includes('parent')) {
+        return null;
+      }
+      if (this.getCurrentQueryString().includes('collection_id')) {
```

Comment:
Use `this.getUrlParams().hasOwnProperty('collection_id')`

Reply:
Done.